manifest/default.xml

```diff
@@ -80,7 +80,7 @@
 
   <project name="osext" path="godeps/src/github.com/kardianos/osext" remote="couchbasedeps" revision="29ae4ffbc9a6fe9fb2bc5029050ce6996ea1d3bc"/>
 
-  <project name="go-oidc" path="godeps/src/github.com/coreos/go-oidc" remote="couchbasedeps" revision="5aa9381f6e998aa16cc96b4347d33dcc29792864"/>
+  <project name="go-oidc" path="godeps/src/github.com/coreos/go-oidc" remote="couchbasedeps" revision="8d771559cf6e5111c9b9159810d0e4538e7cdc82"/>
```

Comment:
All of our third-party dependencies are mirrored here: https://github.com/couchbasedeps/

We'll need to update go-oidc here to include this commit.

Let's run through this in sync-up and put it into the dev knowledge base doc at the same time.

Reply:
Thank you Ben for pointing this out. The go-oidc repo has been mirrored and updated to latest revision by running the Jenkins job: http://server.jenkins.couchbase.com/job/update-couchbasedep/85